File: js/ejer33.js
//solicitar numero y mostrar su tabla de multiplicar del 1 al 10
let numero = parseInt(prompt("Ingrese un numero para mostrar su tabla de multiplicar del 1 al 10:"));   
document.write("<h2>Tabla de multiplicar del " + numero + "</h2>");
for (let i = 1; i <= 10; i++) {
    let resultado = numero * i;
    document.write(numero + " x " + i + " = " + resultado + "<br>");
}

File: python/ejer33.py
#  solicitar numero y mostrar su tabla de multiplicar del 1 al 10
numero = int(input("Ingrese un numero para mostrar su tabla de multiplicar del 1 al 10: "))
print(f"Tabla de multiplicar del {numero}")
for i in range(1, 11):
    resultado = numero * i
    print(f"{numero} x {i} = {resultado}")

        
